products/memgraph/v0.15.0/CHANGELOG.md

```diff
@@ -0,0 +1,231 @@
+# Change Log
+
```

Comment:
Add v0.15.0 here and add your function. It would also be good to add other features that were added in the meantime.